x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/AutoConfigureNode.java

```diff
@@ -635,7 +636,11 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
                             bw.newLine();
                             bw.write("# and all the subsequent nodes should be added via the node enrollment flow");
                             bw.newLine();
-                            bw.write(ClusterBootstrapService.INITIAL_MASTER_NODES_SETTING.getKey() + ": [\"${HOSTNAME}\"]");
+                            bw.write(
+                                ClusterBootstrapService.INITIAL_MASTER_NODES_SETTING.getKey()
```

Comment:
I would adjust at least one of the packaging tests to the effect that the 'node.name' setting is randomly set, and the auto-configurations honors it (ie the initial node starts and forms a cluster).

Reply:
Thanks that made total sense, I added https://github.com/elastic/elasticsearch/pull/81007/files#diff-fcbff32962b19f8c6dfd6bd30c53705f4745d328bc8797382d19e1aaa18c57d4R263